libpcsxcore/psxbios.c

```diff
@@ -3271,6 +3275,7 @@ void psxBiosException() {
 void psxBiosFreeze(int Mode) {
 	u32 base = 0x40000;
 
+	pad_stopped = 0;
```

Comment:
Saving or loading a savestate now makes the BIOS act as if psxBios_StartPAD() was called. Should at least only do it on state load and set it according to pad_buf1 being set to NULL perhaps.

Reply:
Would it be okay if i do bfreezel(&pad_stopped); instead ?